packages/main/src/components/Toolbar/index.tsx

```diff
@@ -307,10 +307,12 @@ const Toolbar = forwardRef<HTMLDivElement, ToolbarPropTypes>((props, ref) => {
   };
 
   const prevChildren = useRef(flatChildren);
-  const debouncedOverflowChange = useRef(debounce(onOverflowChange, 60));
+  const debouncedOverflowChange = useRef();
```

Comment:
Types are not inferred anymore, please add them.

```suggestion
  const debouncedOverflowChange = useRef<ToolbarPropTypes['onOverflowChange'] & { cancel(): void }>();
```